.github/workflows/main.yml

```diff
@@ -14,13 +14,19 @@ jobs:
         uses: actions/checkout@v2
         with:
           lfs: false
+      - name: build latest runc # https://github.com/actions/virtual-environments/issues/2698
```

Comment:
Can you explicitly write "Remove step when {...} is solved" on both instances of "build latest runc" here?

Reply:
I created #122 to follow up on this PR. I can add a TODO(#122) in both places

Reply:
```
# TODO(#122): Remove step when PR #121 is solved
```

Don't you mean `# TODO(#122): Remove step when Issue #122 is solved`

Reply:
oh yeah, will fix